src/System Application/App/Language/src/Language.Codeunit.al

```diff
@@ -67,6 +67,42 @@ codeunit 43 Language
         exit(LanguageImpl.GetLanguageId(LanguageCode));
     end;
 
+    /// <summary>  
+    /// Sets an override for the language ID.   
+    /// This function is used to override the default or user-selected language ID.   
+    /// Once set, the system will return this language ID for any subsequent calls to `GetLanguageIdOrDefault` method, until the override is cleared.  
```

Comment:
This is not what was discussed. The override should reset after every call. If the override does not reset and an error occurs, the language will remain changed until the user starts a new session or hits another override.

Reply:
Hmm, well. You are right.

Reply:
What if you have multiple calls to this GetLanguageId in a report? Then it will only override on the first, then remaining iterations will be printed in wrong language.

Reply:
as discussed, lets have an overload with and one without reset... and doc the error issue + add telemetry whenever override is used.